src/encoded/static/components/search.js

```diff
@@ -1464,6 +1464,14 @@ export const FacetList = (props) => {
         }
     }, [context.facet_groups, prevFacetGroups]);
 
+    if (props.context.title === 'Immune Cells' && facets.length === 0) {
+        return (
+            <>
+                {props.additionalFacet}
+            </>
+        );
+    }
+
```

Comment:
It sounded like you only needed this for local testing, so among my CR comments, I’d say removing this is the one required change. Nothing in these facet functions should specifically serve just one page’s facets — only a custom facet run through the facet registry should.

Reply:
Found a different solution for this.